### ISSUE TITLE
Hang on gappy media

### ISSUE THREAD

**Issue by Zorig** [google/shaka-player#656]
I am using shaka 2.0.2 to play some DRM song from my server. In my case some of my music play fine. But sometimes it's not even start playing. When i see network on chrome(55) that is not start playing song, It is not sending further request to server. I am testing it on Ubuntu16.04 with Chrome(55.0.2883.87), Firefox(50.1.0) Here is sending request network
![screenshot from 2017-01-14 17 41 45](https://cloud.githubusercontent.com/assets/1277672/21954012/e7d2332e-da80-11e6-8552-28d4d5c4f163.png)

and here is the one is not playing
![screenshot from 2017-01-14 17 37 49](https://cloud.githubusercontent.com/assets/1277672/21953998/87a7fc0e-da80-11e6-9210-9a87c273231d.png)

how can i fix it. According to initiator it's related to shaka or shaka config. I set retryParameters with following:

```
this.get('player').configure({
                        abr: {
                            enabled: false
                        },
                        streaming: {
                            retryParameters: {
                                timeout: 0, // timeout in ms, after which we abort a request; 0 means never
                                maxAttempts: 5, // the maximum number of requests before we fail
                                baseDelay: 1500, // the base delay in ms between retries
                                backoffFactor: 2, // the multiplicative backoff factor between retries
                                fuzzFactor: 0.5, // the fuzz factor to apply to each retry delay
                            }
                        },
                    });
```

**Comment by joeyparrish** [comment]
Hi Zorig,

I will need some more information from you to debug this.  In future, please use our issue template, which asks all the questions that we need answered before we can help you.  Here are the ones I still need answers to:

- Can you reproduce the issue with our latest release version (v2.0.3)?

- Can you reproduce the issue with the latest code from `master`?

- Are you using the demo app or your own custom app?
  - If custom app, can you reproduce the issue using our demo app?

- What browser and OS are you using?

- What content did you load?  If custom asset, what manifest and license server URIs?

- How did you interact with the content, if at all?

**Comment by Zorig** [comment]
Hi @joeyparrish,
Ok i got it. So here is the answers.

- I just tested it with latest version (v2.0.3), still some of my songs are not playing
- Even with latest code from master branch shows this bug.
- I am using it on custom app. Ember(2.10)
    - I was using shaka(v2.0.0-beta2) it was working perfectly fine.
- Testing browser is chrome(55), firefox(50) on linux(ubuntu 16.04), mac osx sierra
- I load songs from my server through widevine expressPlay (DRM)
- With album, or song play in shaka player.

Plus i got robustness level warning in console. Hence i thought it's because of i did not set robustness level. Then i set robustness level on it still warning exists

**Comment by joeyparrish** [comment]
The robustness warning can be ignored in this case.  It is generated by Chrome, not by Shaka Player.

Can you please share manifest and license server URIs so that we can reproduce your issue?  There's nothing we can do if we can't reproduce this.  If you can't share them publicly, please send me a private email.  My address can be found in the CONTRIBUTORS file.

**Comment by Zorig** [comment]
i think this is what happened to me #657, little problem is my content's link is only one time(it generates random token url for content in widevine service) so even if i sent you manifest it wont work. If it's okay i'll send you web-app login credentials and URL to your email. This is the only way you can check issue in real-time

**Comment by dulmandakh** [comment]
Hi @joeyparrish, I sent you an email with manifest and license server URI.

Thank you

**Comment by joeyparrish** [comment]
As discussed offline, this appears to be an issue with gaps in the media.  If you can't get your media to start at PTS=DTS=0, you can wait for our solution to #555, which should solve this situation as well.

For now, I'm closing this as a duplicate of #555, but please let us know if you have any further questions.  Thanks!
